starport-core/src/main/scala/com/krux/starport/SummarizeUnmanagedPipelines.scala

```diff
@@ -0,0 +1,23 @@
+package com.krux.starport
+
+import com.krux.starport.util.AwsDataPipeline
+
+object SummarizeUnmanagedPipelines extends StarportActivity {
+
+  def run(): Unit = {
+    val pipelinesInAws = AwsDataPipeline.listPipelineIds() -- inConsoleManagedPipelineIds()
+    val pipelineStatuses = AwsDataPipeline.describePipeline(pipelinesInAws.toSeq: _*)
+
+    logger.info("Unmanaged Pipeline Count By State:")
+    pipelineStatuses.groupBy(_._2.pipelineState).mapValues(_.size).toSeq.sortBy(_._2)(Ordering[Int].reverse)
+      .foreach { case (state, count) => logger.info(s"${state.getOrElse("Unknown")} -> $count") }
```

Comment:
would it be helpful to also show by date?

Reply:
@anniezhang77 - yes. I updated the PR. Can you take another look?